modules/distr-tracing-install-jaeger-operator.adoc

```diff
@@ -49,4 +44,4 @@ The *Manual* approval strategy requires a user with appropriate credentials to a
 
 . Navigate to *Operators* -> *Installed Operators*.
 
-. On the *Installed Operators* page, select the `openshift-operators` project. Wait until you see that the {JaegerName} Operator shows a status of "Succeeded" before continuing.
+. On the *Installed Operators* page, select the `openshift-operators` project. Wait until you see that the {JaegerOperator} Operator shows a status of "Succeeded" before continuing.
```

Comment:
I would use bold instead of quotes for Succeeded

Reply:
Good catch. I was only focused on updating the Operator names here.
These files where added when the engineering team had no technical writer assigned to their project, but I'm reluctant to go through it all because they plan to drop this product and all of its docs very soon.